winter-website-contact-form.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE HTML>
+
+<html lang="en-GB">
+
+<head>
+	
+	<div class="navBar">
+				<a class="navLink" 				href="file:///Users/ioacademy/Documents/winter-website/winter-website.html#">Home</a>
```

Comment:
whitespace not needed here

Reply:
also mega big deal is you arent using relative paths for your different pages so its not loading them up for me.
they all need to be changed to just their filename as they're in the same folder

ie "winter-website-what-i-do.html"

you will need to do the same thing for your style sheets too if they arent relative